iosApp/Podfile

```diff
@@ -1,5 +1,54 @@
-target 'iosApp' do
-  use_frameworks!
-  platform :ios, '14.1'
+platform :ios, '11.0'
+use_frameworks!
+inhibit_all_warnings!
+
+def testing_pods
+  pod 'Quick'
+  pod 'Nimble'
+  # TODO: Remove or update the version of `1.8.0` to the newest version (not 1.8.1) when init a new project.
+  # Currently, there is a bug on `1.8.1` - the newest version.
+  pod 'Sourcery', '1.8.0'
+  pod 'SwiftFormat/CLI'
+end
+
+target 'Survey' do
+  # UI
+  pod 'Kingfisher'
+  pod 'SnapKit'
+
+  # Storage
+  pod 'KeychainAccess'
+
+  # Tools
+  pod 'Firebase/Crashlytics'
+  pod 'IQKeyboardManagerSwift'
+  pod 'NimbleExtension', :git => 'https://github.com/nimblehq/NimbleExtension', :branch => 'master'
+  pod 'R.swift'
+  pod 'Resolver' # Needs Cocoapods on iOS 11 to support Resolver
+
+  # Development
+  pod 'SwiftLint'
+  pod 'Wormholy', :configurations => ['Debug Staging', 'Debug Production']
+
+  # KMM
+  platform :ios, '14.0'
   pod 'shared', :path => '../shared'
```

Comment:
If you want to update to
`pod 'Shared', :path => '../shared'`

You can update in `shared/build.gradle.kts` [like this](https://github.com/suho/kmm-ic/pull/86/files#diff-cf9ae3b1eb7273a914f22b1e6d22d8447f88df0e7245a5b856fb751e507b979bR23)

So we can use `import Shared` in the iOS project

Reply:
This change is having issue on the CI. Will try to open a new PR for this name change specifically.